src/components/wallet/PageTransactions.vue

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prettier/prettier */
```

Comment:
why is this in here?

Reply:
I have absolutely no idea. :) I didn't put it there intentionally.